declare function catchErrors<T extends Function>(done: jest.DoneCallback, callback: T): T;
